.github/workflows/deploy-to-production.yml

```
@@ -80,26 +90,28 @@ jobs:
+          DIGESTS+="${{ env.REGISTRY_IMAGE }}:${PLATFORM}@${DIGEST} "
           done
+          
+          docker buildx imagetools create --tag "${{ env.REGISTRY_IMAGE }}:${{ matrix.tag }}" "$DIGESTS"
 
-      - name: Inspect manifest lists 🕵️
-        run: |
-          for tag in "${{ github.event.release.tag_name }}" "latest"; do
-            docker buildx imagetools inspect "${{ env.REGISTRY_IMAGE }}:$tag"
-          done
+      - name: Inspect manifest list 🕵️
+        run: docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:${{ matrix.tag }}
```

actionlint

property "tag" is not defined in object type {}

(expression)